lib/cog/chat/slack/template_processor.ex

```diff
@@ -4,14 +4,22 @@ defmodule Cog.Chat.Slack.TemplateProcessor do
   alias Cog.Util.Colors
 
   @markdown_fields ["author", "text", "title", "pretext"]
+  @too_long_for_message 256
 
-  def render(directives) do
-    {text, attachments} = directives
-                          |> Enum.map(&process_directive/1) # Convert all Greenbar directives into their Slack forms
-                          |> List.flatten                   # Flatten nested lists into a single list
-                          |> consolidate_outputs({"", []})  # Separate message text and attachments into separate lists
-    attachments = Enum.map(attachments, &finalize_attachment/1) # Final attachment post-processing for Slack
-    {text, attachments}
+  def render(directives, convert_long_text \\ true)
+
+  def render(directives, true) do
+    {text, attachments} = render_directives(directives)
+    # Message is too long convert it to an attachment
+    if String.length(text) > @too_long_for_message do
+      {"", [%{"text" => text, "fallback" => text, "color" => Colors.name_to_hex("operableblue"),
+              "title" => "Pipeline Result", "mrkdwn_in" => ["text", "title"]}]}
```

Comment:
This still swallows any attachments that may have been present... and still probably behaves weirdly when the children of an attachment render as a too-long string.

Are we just going to roll with this for now, since we have the ability to sidestep it for the error attachments and see how it shakes out with wider usage?